Assignment5/simulateCarDrive.js

```diff
@@ -0,0 +1,108 @@
+const {
+  CarCrashError,
+  SpeedingError,
+  EngineFailureError,
+  FlatTireError,
+} = require("./errors");
+
+function driveCar() {
+  try {
+    console.log("Starting the car...");
+    startCar();
+
+    console.log("Driving safely...");
+    driveSafely();
+
+    console.log("Checking speed...");
+    checkSpeed();
+
+    console.log("Parking the car...");
+    parkCar();
+  } catch (error) {
```

Comment:
Can we have multiple catch blocks instead of if-else?

Reply:
we can not use multiple catch blocks in js